tools/seedconv/main.go

```diff
@@ -0,0 +1,52 @@
+package main
```

Comment:
Instead of add a new CLI move this into a sub command of `tfuser id`

Reply:
True, was an external tool to make things easier to test, I can now move it.